torch/_inductor/codegen/wrapper.py

```diff
@@ -353,6 +356,53 @@ def call(args):
                 )
             V.graph.sizevars.codegen(self.wrapper_call, V.graph.graph_inputs)
 
+    def write_mkldnn_param_gen_utils(self):
```

Comment:
I don't think this caching based on ID will work with dynamic shapes.

Reply:
Yes, here I assume TorchDynamo will guard the input shape change and invoke TorchInductor to re-generate new sub-graph code (a new python wrapper code, together with some included cpp kernels) which will then include a new param cache for the new shapes. Can we take this assumption? Or any suggestions?